### PR TITLE
Fix Travis after_success

### PR DESCRIPTION
**Pull request by llogiq**
This should only run on Linux.

cc @mcarton

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,9 +43,10 @@ script:
  - set +e
 
 after_success:
-- ./.github/deploy.sh
-# trigger rebuild of the clippy-service, to keep it up to date with clippy itself
 - |
+- if [ $(uname) == Linux ]; then
+    ./.github/deploy.sh
+# trigger rebuild of the clippy-service, to keep it up to date with clippy itself
     #!/bin/bash
```
Comment (mcarton) [inline .travis.yml]: This needs to be moved before the `if` now.

Comment (llogiq) [inline .travis.yml]: Do we need .github/deploy.sh for both OSX and Linux?

Comment (llogiq) [inline .travis.yml]: Or do you mean the `#!/bin/bash`?

Comment (mcarton) [inline .travis.yml]: I meant `#!/bin/bash` (but I don't know if it is even necessary).

**Pull request merged**